test/src/401-ducc-convert_singularity/main

```diff
@@ -4,10 +4,15 @@ cvmfs_test_suites="ducc"
 
 DOCKER_TEST_IMAGE="https://registry.hub.docker.com/library/centos:centos7"
 
+ducc_test_401_clean_up() {
+  sudo cvmfs_server rmfs -f $CVMFS_TEST_REPO
+}
 
 cvmfs_run_test() {
+  trap ducc_test_401_clean_up EXIT HUP INT TERM
+
   echo "*** creating empty repo ***"
-  create_empty_repo $CVMFS_TEST_REPO $CVMFS_TEST_USER || return 1
+  create_empty_repo $CVMFS_TEST_REPO $USER || return 1
```

Comment:
Why `$USER` instead of `$CVMFS_TEST_USER`

Reply:
DUCC expect to be run by the user who owns the repository.

Reply:
I see, thanks